docs/source/conditions.rst

```diff
@@ -498,3 +498,54 @@ every status check that is expected, e.g.:
   that have ``build`` in their name (see point 1. and 2.above).
 
 Such conditions won't do what you want them to do.
+
+About Branch Protection
+~~~~~~~~~~~~~~~~~~~~~~~
+
+GitHub allows you to enable branch protection settings on your repository.
+For some of the branch protection parameters, listed as follows, Mergify is going to generate
+corresponding conditions in pull request and queue rules.
```

Comment:
```suggestion
GitHub allows you to enable branch protection settings on your repository.
For some of the branch protection parameters, listed below, Mergify is going to generate
corresponding conditions in pull request and queue rules.
```

Where is the list? 😄

Reply:
the list is in the png image just below, it's a screen of the branch protection settings list in GitHub. I can eventually wright a list instead ! 

+ I modified the .png files and ran `pngcrush` on them as you said !

Reply:
I'd rather write it in text: images is not great to read, and you cannot search (C-f) in it :(

Reply:
Replaced the png with a list containing only the handled branch protection settings and changed some grammar like suggested